src/components/React/ControlsPanel.js

```diff
@@ -33,7 +33,7 @@ export default function ControlPanel(props) {
           <ToggleButtonGroup
             type="radio"
             name="measurement"
-            value={options.measurement}
+            value={selection.measurement}
```

Comment:
@RobertGemmaJr as a follow on PR, consider modifying the config so you also have the human name in it, allowing you to automate the toggle button renders

Reply:
Yeah the only issue is that we'd have to change the `png` file names because that's how we generate  the path. That would definitely work though

Reply:
not necessarily, you could change the config to look something like
```json
{
  "season": [{"value": "salt", "name": "Salinity"},...]
  ...
}
```

This also opens the door for adding more metadata  to those values for auto-composing some text blurb for each thing.

Reply:
Ouu very smart I'll definitely add it in a later pr